src/aws/lambda-encode/src/encodeTimedTextTrack.js

```diff
@@ -40,10 +40,22 @@ module.exports = async (objectKey, sourceBucket, filename) => {
       // Transform the source key to the format expected for destination keys:
       // 630dfaaa-8b1c-4d2e-b708-c9a2d715cf59/timedtexttrack/dba1512e-d0b3-40cc-ae44-722fbe8cba6a/1542967735_fr
       // 👆 becomes 👇
-      // 630dfaaa-8b1c-4d2e-b708-c9a2d715cf59/timedtext/1542967735_fr
+      // 630dfaaa-8b1c-4d2e-b708-c9a2d715cf59/timedtext/1542967735_fr.vtt
       Key: `${objectKey.replace(/\/timedtexttrack\/.*\//, '/timedtext/')}.vtt`,
     })
     .promise();
+
+  await s3.
+    copyObject({
+      Bucket: destinationBucket,
+      Key: `${objectKey.replace(/\/timedtexttrack\/.*\//, '/timedtext/')}`,
+      CopySource: `${sourceBucket}/${objectKey}`
+    })
+    .promise();
+
+  return Promise.resolve(
+    subsrt.detect(timedTextFile.Body.toString())
```

Comment:
Add comment with examples of returned values? what happens if I upload a file that is not recognized?

Reply:
the lambda will faill before this line. To encode and/or convert a timed text track the lib needs first to detect the format. I can add one more tryc/catch but I thought it is useless here.

Reply:
:+1: